src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInMdx.test.ts

```diff
@@ -0,0 +1,141 @@
+import { dataModelsForTests } from "@activeviam/data-model-5.0";
+import { MdxSelect, parse, stringify } from "@activeviam/mdx-5.0";
+import { migrateCalculatedMeasuresInMdx } from "./migrateCalculatedMeasuresInMdx";
+
+const mdxSelectWithNoCalculatedMeasures: MdxSelect = parse(
+  "SELECT NON EMPTY Hierarchize(Descendants({[Geography].[City].[AllMember]}, 1, SELF_AND_BEFORE)) ON ROWS, NON EMPTY {[Measures].[contributors.COUNT]} ON COLUMNS FROM [EquityDerivativesCube] CELL PROPERTIES VALUE, FORMATTED_VALUE, BACK_COLOR, FORE_COLOR, FONT_FLAGS",
```

Comment:
I meant that it would be nice to have the indented version here too, which is what I was thought could be more of a hassle 😅

Reply:
Oops, sorry 😆. I've indented this now too